### PR TITLE
Show Toolkit Customization Syntax in GPU Operator installation

### PR DESCRIPTION
**Pull request by geekloper**
This PR aims to enhance the documentation by adding instructions for passing options to the GPU Operator during installation. 
Please review the changes and provide feedback.

### CODE REVIEW

**Comment by mikemckiernan** [comment]
@geekloper, thank you for the PR.  If I read correctly, this PR adds a sample command that uses the default values.  The Rancher and MicroK8s sections that are after your update are similar, but have the advantage of showing deployment-specific values.  I feel like there's benefit when the example solves a specific challenge.  Please let me know if I'm overlooking some other benefit.

**Comment by geekloper** [comment]
Thanks for your feedback. 
Well, I noticed a documentation gap for Containerd setups and thought it worth mentioning, as setting Helm values can sometimes be tricky. Aimed to mirror clarity provided for MicroK8s and Rancher. If it doesn't align with documentation goals, i’m okay with closing the PR. PR 😉

File: gpu-operator/getting-started.rst [hunk 1]
```diff
@@ -406,6 +406,21 @@ options are used with the container-toolkit deployed with GPU Operator:
       - name: CONTAINERD_SET_AS_DEFAULT
         value: true
 
+These options can be passed to GPU Operator during install time as below.
```
Comment (mikemckiernan) [inline gpu-operator/getting-started.rst:409]: I agree that the syntax for the options can be tricky.  Do you mind revising the intro statement so the purpose is to clarify the syntax?  Something like the foll?

If you need to specify custom values, refer to the following sample command for the syntax:

**Comment by mikemckiernan** [comment]
@geekloper, can you squash your commits and sign?

**Comment by geekloper** [comment]
@mikemckiernan it's done ✅

**Comment by mikemckiernan** [comment]
Thank you again @geekloper!

**Pull request merged**